azext_iot/_help.py

```diff
@@ -298,48 +298,81 @@
     examples:
     - name: Get the parent device of the specified device.
       text: >
-        az iot hub device-identity get-parent -d {non_edge_device_id} -n {iothub_name}
+        az iot hub device-identity get-parent -d {device_id} -n {iothub_name}
 """
 
 helps[
     "iot hub device-identity set-parent"
 ] = """
     type: command
-    short-summary: Set the parent device of the specified non-edge device.
+    short-summary: Set the parent device of the specified device.
     examples:
-    - name: Set the parent device of the specified non-edge device.
+    - name: Set the parent device of the specified device.
       text: >
-        az iot hub device-identity set-parent -d {non_edge_device_id} --pd {edge_device_id} -n {iothub_name}
-    - name: Set the parent device of the specified non-edge device irrespectively the non-edge device is
+        az iot hub device-identity set-parent -d {device_id} --pd {edge_device_id} -n {iothub_name}
+    - name: Set the parent device of the specified device irrespectively the device is
             already a child of other edge device.
       text: >
-        az iot hub device-identity set-parent -d {non_edge_device_id} --pd {edge_device_id} --force -n {iothub_name}
+        az iot hub device-identity set-parent -d {device_id} --pd {edge_device_id} --force -n {iothub_name}
+"""
+
+helps[
+    "iot hub device-identity parent"
+] = """
+    type: group
+    short-summary: Manage IoT device\'s parent device.
+"""
+
+helps[
+    "iot hub device-identity parent show"
+] = """
+    type: command
+    short-summary: Get the parent device of the specified device.
+    examples:
+    - name: Get the parent device of the specified device.
+      text: >
+        az iot hub device-identity parent show -d {device_id} -n {iothub_name}
+"""
+
+helps[
+    "iot hub device-identity parent set"
+] = """
+    type: command
+    short-summary: Set the parent device of the specified device.
+    examples:
+    - name: Set the parent device of the specified device.
+      text: >
+        az iot hub device-identity parent set -d {device_id} --pd {edge_device_id} -n {iothub_name}
+    - name: Set the parent device of the specified device irrespectively the device is
```

Comment:
This use of `irrespectively` (which was there before) doesn't sound right. Consider an improvement like:

"Set the parent device of a specified device regardless of if the device already has a parent."